Source/mpq/mpq_writer.hpp

```diff
@@ -14,14 +14,10 @@
 namespace devilution {
 class MpqWriter {
 public:
-	bool Open(const char *path);
-
-	bool Close(bool clearTables = true);
-
-	~MpqWriter()
-	{
-		Close();
-	}
+	MpqWriter(const char *path);

```

Comment:
Single-argument constructors should be marked `explicit`

Reply:
Fixed 🙂